examples/3d/camera_sub_view.rs

```diff
@@ -0,0 +1,267 @@
+//! Renders multiple cameras with different sub view efffects.
```

Comment:
This needs a less jargon-y explanation: what's a "sub view", and why might a user care about this?

Reply:
I added mentions of some possible use cases, however the screen shake use case is not covered by my example. Is it fine to leave it in?